metals/src/main/scala/scala/meta/internal/metals/callHierarchy/CallHierarchyProvider.scala

```diff
@@ -103,8 +107,16 @@ final case class CallHierarchyProvider(
       params: CallHierarchyIncomingCallsParams,
       token: CancelToken,
   )(implicit ec: ExecutionContext): Future[List[CallHierarchyIncomingCall]] = {
-    val info = getInfo(params.getItem.getData)
-
+    val parents = supermethods
+      .getSuperMethodHierarchySymbols(
```

Comment:
Maybe we could create an additiona `getSuperMethodHierarchySymbols` method that would take in AbsolutePath and Position? We wouldn't need to create TextDocumentPositionParams